platform/app/pluginConfig.json

```diff
@@ -68,12 +68,10 @@
     },
     {
       "packageName": "@ohif/mode-test",
-      "default": false,
```

Comment:
It should be possible to load the modes without them being defaults by configuring them in the app config - that is the preferred way to add these, rather than having them get listed all the time.

Reply:
Yeah maybe another PR